RNTester/RNTester.xcodeproj/project.pbxproj

```diff
@@ -191,6 +191,52 @@
 		9F465A02224EB106005BEE04 /* libRCTWebSocket.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 9FD21CF92242EFA400E7F88E /* libRCTWebSocket.a */; };
 		9F465A03224EB10E005BEE04 /* libReact.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 9FD21D122242EFA400E7F88E /* libReact.a */; };
 		9F465A05224EB177005BEE04 /* libRCTTest-macOS.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 18DF57271EFA056600BF4666 /* libRCTTest-macOS.a */; };
+		9F6168B9225EA318008F89F1 /* libRCTWebSocket.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 139FDED91B0651EA00C62182 /* libRCTWebSocket.a */; };
```

Comment:
Maybe I'm misreading this, but why would static libs (.a's) go in the frameworks folder?

Reply:
That's what Xcode calls libraries/frameworks in the "Link Binary with Libraries" build phase. See attached image and note the label at the bottom

<img width="679" alt="Screen Shot 2019-04-12 at 10 25 46 AM" src="https://user-images.githubusercontent.com/48299693/56055039-5c89bf00-5d0d-11e9-9561-4eb4c170034a.png">